package.json

```diff
@@ -16,22 +16,23 @@
   },
   "dependencies": {
     "@docusaurus/core": "3.3.2",
-    "@docusaurus/preset-classic": "3.3.2",
     "@docusaurus/plugin-pwa": "3.3.2",
+    "@docusaurus/preset-classic": "3.3.2",
+    "@docusaurus/theme-mermaid": "3.3.2",
```

Comment:
I checked out this change and ran `docker compse up --build` to build an image with the new dependencies but got this error. Not sure exactly what the cause is:
```
ozone-site-site-1  | 
ozone-site-site-1  | > doc-site@0.1.0 start /ozone-site
ozone-site-site-1  | > docusaurus start --port 3001 "--host" "0.0.0.0"
ozone-site-site-1  | 
ozone-site-site-1  | [INFO] Starting the development server...
ozone-site-site-1  | 
ozone-site-site-1  | [ERROR] Error: Docusaurus was unable to resolve the "@docusaurus/theme-mermaid" theme. Make sure one of the following packages are installed:
ozone-site-site-1  | - @docusaurus/theme-mermaid
ozone-site-site-1  | - @docusaurus/docusaurus-theme-theme-mermaid
ozone-site-site-1  |     at resolveModuleName (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/plugins/moduleShorthand.js:41:15)
ozone-site-site-1  |     at normalizeShorthand (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/plugins/configs.js:76:60)
ozone-site-site-1  |     at /ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/plugins/configs.js:90:63
ozone-site-site-1  |     at Array.map (<anonymous>)
ozone-site-site-1  |     at loadPluginConfigs (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/plugins/configs.js:90:48)
ozone-site-site-1  |     at async initPlugins (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/plugins/init.js:42:27)
ozone-site-site-1  |     at async /ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/plugins/plugins.js:136:36
ozone-site-site-1  |     at async loadSite (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/server/site.js:127:45)
ozone-site-site-1  |     at async createReloadableSite (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/commands/start/utils.js:46:16)
ozone-site-site-1  |     at async Command.start (/ozone-site/node_modules/.pnpm/@docusaurus+core@3.3.2_@docusaurus+types@3.3.2_react-dom@18.2.0_react@18.2.0_typescript@5.4.5/node_modules/@docusaurus/core/lib/commands/start/start.js:22:28)
ozone-site-site-1  | [INFO] Docusaurus version: 3.3.2
ozone-site-site-1  | Node version: v20.18.0
ozone-site-site-1  |  ELIFECYCLE  Command failed with exit code 1.
ozone-site-site-1 exited with code 1
```

Reply:
I saw that I faced the same problem after uninstalling docusaurus/theme-mermaid.
I was able to resolve it by reinstalling the package again on my mac. 
Next, I ran `docker compse up --build` and then ran it with no issues.